jira/client.py

```diff
@@ -4516,13 +4516,27 @@ def sprints(
             self.AGILE_BASE_URL,
         )
 
-    def sprints_by_name(self, id, extended=False):
+    def sprints_by_name(self, id, extended=False, state=None):
```

Comment:
shall we also add typehints here?

```suggestion
    def sprints_by_name(self, id: Union[str,int], extended: bool = False, state: str = None) -> Dict[str, Dict[str, Any]]:
```

the output type is a bit weird for this function (ie. could spit out the sprint object itself?)... but probably beyond this PR